src/plasma/status.h

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
```

Comment:
This code is actually from arrow (which presumably took it from leveldb) and will be removed from our codebase after the arrow merge